MekHQ/src/mekhq/gui/view/ContractSummaryPanel.java

```diff
@@ -226,7 +226,9 @@ public void mouseClicked(MouseEvent e) {
             mainPanel.add(lblDistance, gridBagConstraintsLabels);
 
             JumpPath path = campaign.calculateJumpPath(campaign.getCurrentSystem(), contract.getSystem());
-            int days = (int) Math.ceil(path.getTotalTime(Utilities.getDateTimeDay(contract.getStartDate()),
+
+            int days = (int) Math.ceil(path.getTotalTime(
+                    contract.getStartDate().toInstant().atZone(ZoneId.systemDefault()).toLocalDate(),
```

Comment:
This seems more awkward than the previous syntax, is there any way to make it less, uh...

Reply:
This is temporary, until I switch contracts to LocalDate

Reply:
Sounds good.